### PR TITLE
implement reverse conversion from UTM/NWU to Solution (lat/lon)

### PR DESCRIPTION
**Pull request by doudou**

### CODE REVIEW

**Comment by doudou** [comment]
@leifole ... would you mind having a look at this ?

**Comment by leifole** [comment]
sorry, quite busy atm. Won't find the time in the next two weeks.

**Comment by doudou** [comment]
Damn. Thanks anyway ;-)

File: src/UTMConverter.cpp [hunk 1]
```diff
@@ -9,27 +9,62 @@ UTMConverter::UTMConverter()
...
     , origin(base::Position::Zero())
-    , coTransform(NULL)
+    , utm2latlon(nullptr)
+    , latlon2utm(nullptr)
 {
     createCoTransform();
 }
 
-void UTMConverter::createCoTransform()
+UTMConverter::UTMConverter(UTMConversionParameters parameters)
```
Comment (g-arjones) [inline src/UTMConverter.cpp]: Why pass by value?

File: src/UTMConverter.cpp [hunk 2]
```diff
@@ -9,27 +9,62 @@ UTMConverter::UTMConverter()
...
 {
-    OGRSpatialReference oSourceSRS;
-    OGRSpatialReference oTargetSRS;
+    createCoTransform();
+}
 
-    oSourceSRS.SetWellKnownGeogCS("WGS84");
-    oTargetSRS.SetWellKnownGeogCS("WGS84");
-    oTargetSRS.SetUTM(this->utm_zone, this->utm_north);
+void UTMConverter::setParameters(UTMConversionParameters parameters)
```
Comment (g-arjones) [inline src/UTMConverter.cpp]: Same

Comment (doudou) [reply src/UTMConverter.cpp]: Fixed (both)

**Pull request merged**